docs/trace/customizing-the-sdk/README.md

```diff
@@ -446,45 +446,34 @@ it is shutdown.
 `TracerProvider`. Only a single `TraceProvider` may exist in an
 `IServiceCollection` \ `IServiceProvider`.
 
-### Dependency injection `TracerProviderBuilder` extension method reference
+### Dependency injection TracerProviderBuilder extension method reference
 
 * `AddInstrumentation<T>`: Adds instrumentation of type `T` into the
   `TracerProvider`.
 
+* `AddInstrumentation<T>(Func<IServiceProvider, T> instrumentationFactory)`:
```

Comment:
do mention that this SP is the one from the app's host? anything to mention for .NET FW users?

Reply:
Tricky to articulate, I took a stab at it on #4129